client/src/pages/SearchSitter/SearchSitter.tsx

```diff
@@ -1,58 +1,88 @@
+import { useEffect, FormEvent, useState, ChangeEvent } from 'react';
 import CssBaseline from '@material-ui/core/CssBaseline';
+import SearchIcon from '@material-ui/icons/Search';
 import Box from '@material-ui/core/Box';
 import Grid from '@material-ui/core/Grid';
+import TextField from '@material-ui/core/TextField';
 import Typography from '@material-ui/core/Typography';
 import useStyles from './useStyles';
-import SearchSitterForm from '../../components/SearchSitterForm/SearchSitterForm';
 import SitterCard from '../../components/SitterCard/SitterCard';
-import { Button } from '@material-ui/core';
-import { useState, ChangeEvent } from 'react';
-import { MaterialUiPickersDate } from '@material-ui/pickers/typings/date';
+import { Profile } from '../../interface/User';
+import { Button, InputAdornment } from '@material-ui/core';
+import { getAllSitterProfiles } from '../../helpers/APICalls/getSitterProfile';
+import { searchProfiles } from '../../helpers/APICalls/searchUsers';
 
-export default function Register(): JSX.Element {
+export default function SearchSitter(): JSX.Element {
   const classes = useStyles();
   const [search, setSearch] = useState<string>('test');
-  const [selectedStartDate, setSelectedStartDate] = useState<Date | null>(new Date());
-  const [selectedEndDate, setSelectedEndDate] = useState<Date | null>(new Date());
+  const [allSitters, setAllSitters] = useState<Profile[]>([]);
 
-  const handleChange = (e: ChangeEvent<HTMLInputElement>, newInputValue: string) => {
-    setSearch(newInputValue);
+  useEffect(() => {
+    getAllSitters();
+  }, []);
+
+  const getAllSitters = () => {
+    getAllSitterProfiles().then((data) => {
+      setAllSitters(data);
+    });
   };
```

Comment:
This can be a lot cleaner and turn into an async/await this also would be following the project overall code convention.